gradle/ballerinaLangLibBuild.gradle

```diff
@@ -66,7 +66,8 @@ class BallerinaLangLibBuildTask extends JavaExec {
         args << distCache
         args << pkgName
         args << skipBootstrap
-
```

Comment:
Shall we exclude this during distribution packaging. And when we create the final doc json.